v0/destinations/google_adwords_offline_conversions/data/TrackCallConversionsConfig.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "destKey": "conversions[0].customVariables.conversionCustomVariable",
+    "sourceKeys": "properties.conversionCustomVariable",
+    "required": false,
+    "metadata": {
+      "type": "toString"
+    }
+  },
+  {
+    "destKey": "conversions[0].customVariables.value",
+    "sourceKeys": "properties.value",
+    "required": false,
+    "metadata": {
+      "type": "toString"
+    }
+  },
```

Comment:
This not the valid wauy and limiting of passing custom variable. We should ask the user from dashboard what are custom variable for click conversions and what are the custom varoable for call conversions and fetch those from properties and then set `customVariables` array inside conversions